packages/@vuepress/theme-blog/layouts/Layout2.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
What is this `Layout2` component about?

Reply:
yeah it seem useless... i didnt pay attention on it